src/components/layouts/GeneralInputComponent.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react'
+
+class GeneralInputComponent extends Component {
```

Comment:
Why is this a class component? What's the need behind making it a class component? Why can't it be a functional component instead?